addons/website/static/src/scss/website.ui.scss

```diff
@@ -146,4 +146,10 @@ body {
         scrollbar-color: $-foreground-color $-background-color;
         scrollbar-width: thin;
     }
+    
+    @include media-breakpoint-down(md) {
+        &:has(.modal-open) {
```

Comment:
This doesn't seem to be applied. Could you try without the `&`? Or maybe something like `body.modal-open` will do the trick?

Reply:
I tested it now and I see it working, I do not know if you meant somewhere else
![modal-open](https://github.com/user-attachments/assets/4f3421c4-3516-4405-8366-47b7a865365a)

Reply:
In Odoo's mobile preview, it seems to work, but if you test this by simply reducing the screen width (or using the mobile previous from the Chrome DevTools), it doesn't seem to work :/

Reply:
That's probably not what we want: overflow: hidden is already added by bootstrap, but not through the modal-open class anymore. If we are to restore bootstrap to do what it did in a distant past, there would need to be a very convincing comment on top of that CSS you are adding.
![image](https://github.com/user-attachments/assets/7b52335a-af55-49b3-b148-1b8c36d32981)

We also have JS logic about opening/closing dialog blocking the scroll *or not* depending on the presence of a backdrop or not. Touching the modal-open class behavior seems risky.